x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/messages/Messages.java

```diff
@@ -154,7 +154,7 @@ public final class Messages {
     public static final String JOB_AUDIT_DATAFEED_NO_DATA = "Datafeed has been retrieving no data for a while";
     public static final String JOB_AUDIT_DATAFEED_MISSING_DATA =
         "Datafeed has missed {0} documents due to ingest latency, latest bucket with missing data is [{1}]."
-            + " Consider increasing query_delay";
+            + " Check if the end-to-end ingestion process is delayed. If it works as expected, consider increasing query_delay.";
```

Comment:
Consider increasing query_delay and investigate the cause of high latency in your ingestion process.